wlroots.syms

```diff
@@ -0,0 +1,53 @@
+WLROOTS_0_0_0 {
+    global:
+        wlr_*;
+        # FIXME: these shouldn't be exported or renamed
+        os_create_anonymous_file;
```

Comment:
This one shouldn't be though

Reply:
Wouldn't this be useful as wlr_create_anonyous_file ?

Reply:
This is exported only because it's used by the screenshooter client. I don't think we want to expose it for compositors.

Reply:
Maybe it would be better to duplicate this function in the screenshooter instead of exporting it.